config/default.yml

```diff
@@ -509,6 +509,15 @@ Rails/ScopeArgs:
   Include:
     - app/models/**/*.rb
 
+Rails/ShortI18n:
+  Description: >-
+                  Use the short form of the I18n methods: I18n.t instead of I18n.translate and
+                  I18n.l instead of I18n.localize.
+  StyleGuide: 'https://rails.rubystyle.guide/#short-i18n'
+  Safe: false
```

Comment:
Is there a particular aspect that you think would make this unsafe? Or just being cautious?

Reply:
User can define and use his custom `translate` and `localize` methods (while with small probability), so I decided to make this cop unsafe.

Reply:
I wrote an idea on https://github.com/rubocop-hq/rubocop-rails/pull/280/files#r446771314 for making this cop safe.